### PR TITLE
app-text/cherrytree: bump version to 0.99.51

### PR DESCRIPTION
**Pull request by flint2**
Signed-off-by: Guillermo Joandet <gjoandet@gmail.com>

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
@flint2 Since this package is currently lacking a maintainer, would you be interested in proxy-maintaining it in ::gentoo? Or maybe maintaining it in ::guru?

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers
https://wiki.gentoo.org/wiki/Project:GURU

File: app-text/cherrytree/cherrytree-0.99.50.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
```
[the remark this answers is not in the document]
Comment (flint2) [reply app-text/cherrytree/cherrytree-0.99.50.ebuild:6]: It builds with python 10 as compat, will change to PYTHON_COMPAT 8..11 in next commit

**Comment by flint2** [comment]
> @flint2 Since this package is currently lacking a maintainer, would you be interested in proxy-maintaining it in ::gentoo? Or maybe maintaining it in ::guru?
> 
> https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers https://wiki.gentoo.org/wiki/Project:GURU
Since package is already in ::gentoo and in active upstream development, I´ve added myself as proxy maintainer

File: app-text/cherrytree/metadata.xml [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+		<name>Guillermo Joandet</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>gnome@gentoo.org</email>
+		<name>GNOME Desktop</name>
+	</maintainer>
```
Comment (AndrewAmmerlaan) [inline app-text/cherrytree/metadata.xml:15]: Has @gentoo/gnome agreed to this?

Comment (flint2) [reply app-text/cherrytree/metadata.xml:15]: No, I will remove it in next commit

Comment (flint2) [reply app-text/cherrytree/metadata.xml]: Done, changed to 8..11

**Comment by thesamesam** [comment]
Can you drop the merge commits? We use rebases in Gentoo (see e.g. https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Repository_settings).

**Comment by flint2** [comment]
> Can you drop the merge commits? We use rebases in Gentoo (see e.g. https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Repository_settings).

Done

**Comment by Flowdalic** [comment]
pkgcheck reports the following issues

```
  PythonAnyMismatchedDepHasVersionCheck: version 0.99.51: BDEPEND: missing check for dev-python/lxml[${PYTHON_USEDEP}] in 'python_gen_any_dep'
  PythonAnyMismatchedDepHasVersionCheck: version 0.99.51: DEPEND: missing check for dev-python/lxml[${PYTHON_USEDEP}] in 'python_check_deps'
  RedirectedUrl: version 0.99.51: HOMEPAGE: permanently redirected: https://www.giuspen.com/cherrytree -> https://www.giuspen.com/cherrytree/
```

I am not sure what exactly triggers PythonAnyMismatchedDepHasVersionCheck, though. Maybe someone else can shed some light into it.

**Comment by flint2** [comment]
Solved removing lxml dependency that were no longer required

**Pull request closed without merging**